waspc/src/Wasp/Project/Common.hs

```diff
@@ -88,7 +88,6 @@ dotWaspInfoFileInGeneratedCodeDir = [relfile|.waspinfo|]
 packageJsonInWaspProjectDir :: Path' (Rel WaspProjectDir) (File PackageJsonFile)
 packageJsonInWaspProjectDir = [relfile|package.json|]
 
--- TODO: The entire tsconfig story is very fragile
```

Comment:
Did we cover this with some sort of an issue?

Reply:
Yes and no.

I wrote the comment back when the code looked differently and left it by accident. Still, there are things we could improve. And those are covered by other issues, such as:
  - https://github.com/wasp-lang/wasp/issues/2390
  - https://github.com/wasp-lang/wasp/issues/2426
  
I created the second one now, so thanks for pointing this out :)